.github/workflows/dotnetcore.yml

```diff
@@ -23,4 +23,4 @@ jobs:
           name: Test-Reporter
           path: "**/test-results.trx"
           reporter: dotnet-trx
-          fail-on-error: true
+          fail-on-error: false
```

Comment:
We DO want to fail on failing tests right?

Reply:
When a test fails, the workflow will fail anyway. This step is just reporting. Dependabot doesn't have privileges to receive the report results. (I'll keep monitoring to confirm this behavior.)